src/main/java/net/sf/jabref/openoffice/OOUtil.java

```diff
@@ -236,7 +239,7 @@ public static void insertTextAtCurrentLocation(XText text, XTextCursor cursor, S
 
         // TODO: the <monospace> tag doesn't work
         /*
-        if (formatting.get(FORMAT_MONOSPACE)) {
+        if (formatting.contains(Formatting.MONOSPACE)) {
```

Comment:
Also this PR is already merged, does this work with newer OO/LO Versions?
Anyone tried this?